qa/pull-tester/rpc-tests.sh

```diff
@@ -16,6 +16,10 @@ for i in "$@"; do
       EXTENDED="true"
       shift
       ;;
+    -specific=*)
```

Comment:
I thinks it's great now to have the flexibility to run one specific test now, I just wonder if we can rename the `-specific` keyword to something else to improve clarity.

I don't have any particular keyword in mind, so I would like to have feedback from other people.

Reply:
Maybe something like `-runsingletest` can be more clear?

Reply:
As a temporary proposal we could use `-runonly`. We could also consider to refactor the script to have a regex parameter to filter the list of tests.

Reply:
Done